plugin/storage/integration/cassandra_test.go

```diff
@@ -65,6 +67,9 @@ func (s *CassandraStorageIntegration) initializeCassandra(t *testing.T) {
 		"--cassandra-archive.password=" + password,
 		"--cassandra-archive.username=" + username,
 	})
+	t.Cleanup(func() {
```

Comment:
the responsibility to close should be on the function that initialized the factory.